com.incquerylabs.instaschema.performance/src/com/incquerylabs/instaschema/mondo/sam/ResultSetRetrievalPhase.xtend

```diff
@@ -17,17 +17,17 @@ class ResultSetRetrievalPhase extends AtomicPhase {
 	}
 	
 	override execute(DataToken token, PhaseResult phaseResult) {
-		val logger = Logger.getLogger("com.incquerylabs.magicdraw.benchmark");
-		val myToken = token as MyDataToken
+		val logger = Logger.getLogger("org.eclipse.viatra.query");
```

Comment:
Do you really want to use the same logger for the thousands of base index warnings and the LS profiling output?